buildSrc/src/main/groovy/nva.publication.api.git-hooks.gradle

```diff
@@ -0,0 +1,26 @@
+plugins {
+    id 'base'
+}
+
+def scriptsPrePushPath = file('.scripts/pre-push')
+def prePushHookPath = file('./.git/hooks/pre-push')
+
+tasks.register('copyPrePushHook', Copy) {
+    from scriptsPrePushPath
+    into prePushHookPath.parentFile
+    onlyIf {
+        !prePushHookPath.exists() || scriptsPrePushPath.text != prePushHookPath.text
+    }
+    doLast {
+        if (!System.properties['os.name'].toLowerCase().contains('windows')) {
```

Comment:
Apparently, Win does not need this…not sure if it can be done via gradle in this way either